Google.Api.Generator.Utils/SourceFileContext.cs

```diff
@@ -198,7 +200,8 @@ private class TypeAliaser : CSharpSyntaxRewriter
                     IReadOnlyCollection<Typ> seenTypes,
                     IEnumerable<Typ> packageTyps,
                     IReadOnlyDictionary<string, string> wellKnownNamespaceAliases,
-                    IReadOnlyCollection<Regex> avoidAliasingNamespaceRegex)
+                    IReadOnlyCollection<Regex> avoidAliasingNamespaceRegex,
+                    IReadOnlyCollection<string> forceAliases)
```

Comment:
Possibly rename to `forcedAliases`? I'd expect `forceAliases` to be Boolean.

Reply:
Yep, done here and everywhere.